tests/yast2_gui/yast2_storage_ng.pm

```diff
@@ -0,0 +1,242 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: YaST2 ...
+# Maintainer: Paolo Stivanin <pstivanin@suse.com>
+
+use base "y2x11test";
+use strict;
+use warnings;
+use testapi;
+use version_utils qw(is_sle is_tumbleweed is_opensuse);
+
+sub add_logical_volume {
+    my ($lvname, $role) = shift;
+    sleep 1;
+    wait_screen_change { send_key "alt-a" };
+    wait_screen_change { type_string "$lvname" };
+    wait_screen_change { send_key "alt-n" };
+    # custom size
+    if (is_sle('<=12-sp4')) {
+        send_key "alt-c";
+    } else {
+        send_key "alt-t";
+    }
+    sleep 1;
```

Comment:
once again - unconditional sleeps are bad. first of all they always wants more :) sooner or later you will need to come with set of PRs increasing this value to 2,3,4,10 seconds . It is waste of time. Also it is a waste of testing time . And there is **always** a way how to do it without unconditional waits

Reply:
what about using `wait_still_screen` ? Do you think it could be a good replacement?